honeybee_energy/writer.py

```diff
@@ -328,39 +326,32 @@ def model_to_idf(model, schedule_directory=None,
     .. code-block:: python
 
         import os
-        from ladybug.futil import write_to_file_by_name
-        from ladybug.designday import DDY
+        from ladybug.futil import write_to_file
         from honeybee.model import Model
         from honeybee.room import Room
+        from honeybee.config import folders
         from honeybee_energy.lib.programtypes import office_program
         from honeybee_energy.idealair import IdealAirSystem
-        from honeybee_energy.simulationparameter import SimulationParameter
+        from honeybee_energy.simulation.parameter import SimulationParameter
 
         # Get input Model
         room = Room.from_box('Tiny House Zone', 5, 10, 3)
         room.properties.energy.program_type = office_program
-        room.properties.energy.hvac = IdealAirSystem()
+        room.properties.energy.add_default_ideal_air()
```

Comment:
Are you thinking to add this for all the other systems too? I think the normal assignment is the better way to go than add a method for default ideal air.

Reply:
No. For all other systems, I'm expecting the user to create their specific HVAC system class instance like so:
```
import TheHVACSystemClass

the_hvac_system_class_instance = TheHVACSystemClass(name='My HVAC')
room.properties.energy.hvac = the_hvac_system_class_instance
```
I just wanted to have a shortcut for the `IdealAirSystem` because they are used so much all over the place and the convenience of one line of code over 3 is pretty helpful.

Reply:
I don't think we should create an exception for ideal air system in this case even if it is 3 lines versus one.

Also I'm not sure why it will be "used so much all over the place". It will be used once as the default system but other than that it is no different from other HVAC systems. I suggest to keep it consistent.

Reply:
It was so nice, though.  Like [here](https://github.com/ladybug-tools/honeybee-grasshopper-core/pull/18/files). It also helped standardize the naming of the default ideal air system (now, we'll have to set the name differently in each of the 3 components there). I mean, nothing will break by not having a standard naming convention, but it was just nice.